package.json

```diff
@@ -1,59 +1,95 @@
 {
-  "name": "visual-cell-app",
+  "name": "base-react-redux-app",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "http://stash.corp.alleninstitute.org/scm/aics/visual-cell.git"
+    "url": ""
   },
   "author": "",
   "license": "ISC",
+  "scripts": {
+    "start": "webpack-dev-server --config ./webpack/webpack.config.js --env.env dev",
+    "gh-build": "GH_BUILD=true webpack --config ./webpack/webpack.config.js --env.env production",
+    "pregh-deploy": "npm run gh-build",
+    "gh-deploy": "gh-pages -d dist",
+    "build:staging": "NODE_ENV=staging webpack --config ./webpack/webpack.config.js  --env.env staging",
+    "build": "NODE_ENV=production webpack --config ./webpack/webpack.config.js  --env.env production",
+    "lint": "eslint src/**/*.{ts,tsx}",
+    "typeCheck": "tsc -p tsconfig.json --noEmit",
+    "update-viewer": "npm i @aics/simularium-viewer@latest",
```

Comment:
update-viewer doesn't belong.  also what is gh-build supposed to do?

Reply:
i removed it and gh-build. gh-deploy will now just do a regular build by default and push it to the gh-pages branch